contracts/ColonyNetwork.sol

```diff
@@ -135,15 +135,15 @@ contract ColonyNetwork is ColonyNetworkStorage {
     EtherRouter etherRouter = new EtherRouter();
     IColony colony = IColony(address(etherRouter));
     address resolverForLatestColonyVersion = colonyVersionResolver[currentColonyVersion]; // ignore-swc-107
-    etherRouter.setResolver(resolverForLatestColonyVersion);
+    etherRouter.setResolver(resolverForLatestColonyVersion); // ignore-swc-113
 
     // Creating new instance of colony's authority
-    ColonyAuthority authority = new ColonyAuthority(address(colony));
+    ColonyAuthority _authority = new ColonyAuthority(address(colony));
 
     DSAuth dsauth = DSAuth(etherRouter);
-    dsauth.setAuthority(authority);
+    dsauth.setAuthority(_authority);
 
-    authority.setOwner(address(etherRouter));
+    _authority.setOwner(address(etherRouter));
```

Comment:
Why this change? In general it seems like we use the leading underscore convention to denote function arguments, so this usage seems inconsistent.

Reply:
This was because `truffle-security` raised a valid warning on 
```
/home/circleci/colonyNetwork/contracts/ColonyNetwork.sol

║ Line     │ Column   │ Type     │ Message                                                │ Rule ID              ║
╟──────────┼──────────┼──────────┼────────────────────────────────────────────────────────┼──────────────────────╢
║ 141      │ 4        │ warning  │ Local variable shadows a state variable.               │ SWC-119              ║`
```
See https://smartcontractsecurity.github.io/SWC-registry/docs/SWC-119

The underscore is indeed used for function arguments although more to denote local state variable, like [here](https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/ReputationMiningCycleRespond.sol#L409) of which function arguments are part of. I'm happy to be more strict with this convention and rename this to maybe `colonyAuthority`?

Reply:
Ah, I see. Yes `colonyAuthority` would be my preference here.

Reply:
ok, changed.